aspnetcore/blazor/globalization-localization.md

```diff
@@ -40,53 +51,165 @@ The preceding field types:
 * Can't contain free-form text.
 * Provide user interaction characteristics based on the browser's implementation.
 
-The following field types have specific formatting requirements and aren't currently supported by Blazor because they aren't supported by all major browsers:
+When using the `date` and `number` field types, specifying a culture with [`@bind:culture`](xref:mvc/views/razor#bindculture) isn't recommended because Blazor provides built-in support to render values in the current culture.
+
+The following field types have specific formatting requirements and aren't currently supported by Blazor because they aren't supported by all of the major browsers:
 
 * `datetime-local`
 * `month`
 * `week`
 
-[`@bind`](xref:mvc/views/razor#bind) supports the `@bind:culture` parameter to provide a <xref:System.Globalization.CultureInfo?displayProperty=fullName> for parsing and formatting a value. Specifying a culture isn't recommended when using the `date` and `number` field types. `date` and `number` have built-in Blazor support that provides the required culture.
+For current browser support of the preceding types, see [Can I use](https://caniuse.com).
 
-## Localization
+## Demonstration component
+
+The following `CultureExample1` component can be used to demonstrate Blazor globalization and localization concepts covered by this article.
+
+`Pages/CultureExample1.razor`:
+
+```razor
+@page "/culture-example-1"
+@using System.Globalization
+
+<h1>Culture Example 1</h1>
+
+<p>
+    <b>CurrentCulture</b>: @CultureInfo.CurrentCulture
```

Comment:
Does `CultureInfo.CurrentUICulture` works too, I remembered there's a limitation in web assembly for related issue, I may search for the tweet that I made long time ago

Reply:
I didn't test but because the PU focused the original text on `CurrentCulture` only I thought that I'd start with this. Then when Pranav looks, he may say that we need to say and show more about `CurrentUICulture`. I also haven't looked at reference source on this. Perhaps, I should go look.

I also didn't try *only* setting `CurrentUICulture` to see what happens. I figured that since the PU favored the original coverage for `CurrentCulture` that that's what they generally want devs to use.

Reply:
PU stand for?

Reply:
"Product Unit" ... Pranav's original content for this topic focuses on `CurrentCulture`, and he never addressed (much, anyway) `CurrentUICulture`. It was just in a couple of his examples. He sets it, but he doesn't say much about it.